infra/modules/apim/apim-backend.bicep

```diff
@@ -33,12 +33,12 @@ resource backend 'Microsoft.ApiManagement/service/backends@2023-03-01-preview' =
 }
 
 resource apimChargeBackApi 'Microsoft.ApiManagement/service/apis@2023-03-01-preview' = {
```

Comment:
Would it make sense to rename the `apimChargeBackApi` as well? Since we are making breaking changes for v1 anyways. There are more variables using `chargeBack` here.

Reply:
I would leave it to later for cleanup